components/ai_chat/resources/page/components/premium_suggestion/index.tsx

```diff
@@ -77,11 +85,20 @@ function PremiumSuggestion(props: PremiumSuggestionProps) {
         </ul>
       </div>
       {!aiChatContext.isMobile && (
-        <div className={styles.priceList}>
-          <button className={styles.priceButton} tabIndex={-1}>
-            <span className={styles.priceButtonLabel}>Monthly</span>
-            <span className={styles.price}>{pricingInfo}</span>
-          </button>
+        <div className={styles.priceListWrapper}>
+          <div className={styles.priceList}>
+            <button className={styles.priceButton} tabIndex={-1}>
+              <div className={styles.bestValueColumn}>
+                <span className={styles.priceButtonLabel}>{getLocale('oneYearLabel')}</span>
+                <Label color='green'>{getLocale('bestValueLabel')}</Label>
+              </div>
+              <span className={styles.price}>{pricingAnnualInfo}</span>
+            </button>
+            <button className={classnames(styles.priceButton, styles.priceButtonMonthly)} tabIndex={-2}>
```

Comment:
`tabindex={-2}`? This will mean that you can't tab to the button? Is that what we want?

If so, I think it'd be more idiomatic to set it to `-1` too

Reply:
Thinking we should probably change these to `<div>` elements.